Makefile

```diff
@@ -141,3 +141,12 @@ packages-sdk: build
 			tar -cvzf $(PROJECT)_$${TAR_VERSION}_$${os}_$${arch}.tar.gz $(PROJECT)_$${os}_$${arch}/; \
 		done; \
 	done
+
+# TODO: remove when https://github.com/src-d/ci/pull/84 is merged
```

Comment:
Could you also please create an issue for this, mentioning 84 ? That way, as soon as it's merged, it will be harder to forget to do this

Reply:
done in https://github.com/src-d/lookout/issues/132, and also another one for gitbase-playground